app/src/main/res/layout/queue_bottom_sheet.xml

```diff
@@ -83,7 +83,6 @@
                     android:id="@+id/dismiss"
                     android:tooltipText="@string/tooltip_dismiss"
                     style="@style/QueueSheetOption"
-                    android:layout_width="0dp"
```

Comment:
Please drop that - that would break the center alignment of the button and its padding.